main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     cat
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat eats
+cat.eat();
+//  action 3: the cat pounces
+cat.pounce();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     bus
+//  action 1: the bus stops
+bus.stop();
+//  action 2: the bus turns left
+bus.turnLeft();
+//  action 3: the bus honks horn
+bus.honkHorn();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:      grass
+//  action 1:  the grass photosynthesizes
+grass.photosynthesize();
+//  action 2:  the grass disperses seed
+grass.disperseSeed();
+//  action 3:  the grass dies
+grass.die();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     bell
+//  action 1: the bell makes a ringing sound
+bell.ring();
+//  action 2: the bell cracks
+bell.crack();
+//  action 3: the bell swings
+bell.swing();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.